src/core/tsi/ssl_transport_security.cc

```diff
@@ -845,6 +858,22 @@ static tsi_result build_alpn_protocol_name_list(
   return TSI_OK;
 }
 
+// This callback is invoked when the CRL has been verified and will soft-fail
+// errors in verification depending on certain error types.
+static int verify_cb(int ok, X509_STORE_CTX* ctx) {
+  int cert_error = X509_STORE_CTX_get_error(ctx);
+  if (cert_error == X509_V_ERR_UNABLE_TO_GET_CRL) {
+    gpr_log(
+        GPR_INFO,
+        "Certificate verification failed to get CRL files. Ignoring error.");
+    return 1;
```

Comment:
Should we be returning `ok` instead of `1` here?

Reply:
For completeness, I didn't actually add this, just moved it further up in the file.
But to answer the question, no, these verification callbacks just return 1 on success and 0 on failure

Reply:
Ah I see. Why did it need to be moved?

If there's a good reason to move it, can we leave it where it is in this PR and move it in a follow-up PR?

Rationale: This CL has some subtleties to it, and we want to avoid making unrelated changes that will obscure the real functional changes.